packages/design-system/src/components/web-components/ds-choice/ds-choice.stories.tsx

```diff
@@ -0,0 +1,154 @@
+    name: {
+      description: "The `input` field's `name` attribute",
+      control: 'text',
+    },
+    type: {
+      description: 'Sets the type to render `checkbox` fields or `radio` buttons',
+      control: { type: 'radio' },
+      options: ['radio', 'checkbox'],
+    },
+    value: {
+      description: 'The `input` `value` attribute',
+      control: 'text',
+    },
+    'error-message': {
+      description: 'Enable the error state by providing an error message',
+      control: 'text',
+    },
+    hint: {
+      description: 'Hint text or HTML',
+      control: 'text',
+    },
+    label: {
+      description: 'Label text or HTML.',
+      control: 'text',
+    },
+    'label-class-name': {
+      description: 'Additional classes to be added to the field label',
+      control: 'text',
+    },
+    'label-id': {
+      description:
+        "A unique `id` to be used on the field label. If one isn't provided, a unique ID will be generated.",
+      control: 'text',
+    },
+    'requirement-label': { control: 'text' },
+  },
+  args: {
+    'checked-children': 'Checked children',
+    'unchecked-children': 'Unchecked children',
+    type: 'checkbox',
+    label: 'I agree to the above terms and conditions',
+    hint: 'This is some additional hint text',
+    'error-message': 'You must agree to the terms and conditions before continuing',
+    'default-checked': false,
+  },
+  parameters: {
+    docs: {
+      page: WebComponentDocTemplate,
+      description: {
+        component:
+          'For information about how and when to use this component, [refer to checkbox documentation](https://design.cms.gov/components/checkbox/) or [refer to radio button documentation](https://design.cms.gov/components/radio/) pages.',
+      },
+      componentEvents: {
+        'ds-change': {
+          description: 'Dispatched whenever the choice `checked` value changes.',
+          eventObjectDescription: (
+            <>
+              <code>event.details.target.value</code> - The <code>value</code> of the selected
+              option
+            </>
+          ),
+        },
+        'ds-blur': {
+          description: 'Dispatched whenever the choice loses focus.',
+        },
+      },
+      underlyingHtmlElements: ['input'],
+    },
+  },
+  decorators: [webComponentDecorator],
+};
+
+const Template = (args) => {
+  useEffect(() => {
+    const onChange = (event) => {
+      action('ds-change')(event);
+    };
+    const onBlur = (event) => {
+      action('ds-blur')(event);
+    };
+    const choice = document.querySelector('ds-choice');
+    choice.addEventListener('ds-change', onChange);
+    choice.addEventListener('ds-blur', onBlur);
+    return () => {
+      choice.removeEventListener('ds-change', onChange);
+      choice.removeEventListener('ds-blur', onBlur);
+    };
+  });
+
+  return (
+    <ds-choice {...args}>
+      <div slot="checked-children">
+        <div className="ds-c-alert">{args['checked-children']}</div>
```

Comment:
for some reason hooking the story up to the args caused the html to be stripped away. if the value is hard-coded it's fine, so this seems like a storybook problem.

Reply:
Update: @zarahzachz wrote a follow-up ticket for adding partial attribute pass-through support.